lib/webhook/event.ex

```diff
@@ -40,132 +39,152 @@ defmodule StarkBank.Event do
   Receive a single notification Event struct previously created in the Stark Bank API by passing its id
 
   ## Parameters (required):
-    - user [Project]: Project struct returned from StarkBank.project().
     - id [string]: struct unique id. ex: "5656565656565656"
 
+  ## Keyword Args:
+    - user [Project] (optional): Project struct returned from StarkBank.project().
+
   ## Return:
     - Event struct with updated attributes
   """
   @spec get(Project.t(), binary) :: {:ok, Event.t()} | {:error, [%Error{}]}
-  def get(%Project{} = user, id) do
```

Comment:
I think you forgot to change the @spec for get, get!, delete and delete! in StarkBank.Event ...

Reply:
Oops. You're right. Fixed